search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/GenerativeQAResponseProcessor.java

```diff
@@ -58,7 +59,8 @@
  */
 @Log4j2
 public class GenerativeQAResponseProcessor extends AbstractProcessor implements SearchResponseProcessor {
-
+    public static String IllegalArgumentMessage =
+        "Please check the provided generative_qa_parameters are complete and non-null(https://opensearch.org/docs/latest/search-plugins/conversational-search/#rag-pipeline). Messages in the memory can not have Null value for input and response";
```

Comment:
I didn't clearly understand this message. What did you mean by complete here? Can we rephrase the message?